Tests/designspaceLib/designspace_test.py

```diff
@@ -788,98 +801,100 @@ def __removeAxesFromDesignSpace(path):
     f.close()
 
 
-@pytest.fixture
-def invalid_designspace():
-    p = "testCheck.designspace"
-    __removeAxesFromDesignSpace(p)
-    yield p
+# @pytest.fixture
+# def invalid_designspace():
+#     p = "testCheck.designspace"
+#     __removeAxesFromDesignSpace(p)
+#     yield p
+
+
+# #@pytest.mark.xfail(reason="The check method requires MutatorMath")
+# def test_check(invalid_designspace, tmpdir):
```

Comment:
hm, this test was silently broken for a while. It was marked as xfail so we didn't notice. But the reason it was failing was that it was the `invalid_designspace` fixture was attempting to read from an unexisting "testCheck.designspace" file. I don't understand the purpose of the test.
I would prefer we don't leave commented-out code rotting. Either we fix the test or we remove it altogether.

Reply:
The test was for missing axes, it should be removed.